uicomponent-compose/other/src/main/java/io/github/droidkaigi/feeder/staff/AboutThisApp.kt

```diff
@@ -47,8 +50,9 @@ fun WhatIsDroidKaigiComponent() {
             style = typography.h6
         )
         Spacer(modifier = Modifier.height(24.dp))
-        // TODO: Replace DroidKaigi Logo
-        Text("ToDo: Replace me to Logo image")
+        Image(
+            painter = painterResource(R.drawable.logo), contentDescription = "DroidKaigi Logo"
```

Comment:
Apparently, it's better to break the parameters, but it's not a big problem and doesn't need to be fixed.
`break each parameter declaration onto its own line`
https://developer.android.com/kotlin/style-guide?hl=en#functions

Reply:
That's the result of applying spotless.

Reply:
Ah, maybe the spotless option isn't enough, thank you for the information! 🙇